app-typescript/components/avatar/avatar-wrapper.tsx

```diff
@@ -14,11 +14,14 @@ interface IPropsAvatarWrapper {
     };
     children: React.ReactNode;
     'data-test-id'?: string;
-    isEmpty?: boolean;
+    emptyLight?: boolean;
     icon?: {
         name: string;
         color?: string;
     };
+    coverageStatus?: {
```

Comment:
You should avoid naming that describes things outside of this repository. Maybe we intend to use it for coverage statuses, but what we're adding is a colored dot that doesn't have any limitations to usage for coverage statuses only. If we're adding a colored dot, let's call it something like that. `dot` or `statusDot` is what I'd use.

Since we want to apply it to a coverage status, and coverage status is visualized through an `icon` prop, I think it would be easier to understand the meaning if the dot was on top of the icon, not on top of the user picture. When it's on user picture it appears as it is user status like.

What happened by the way with the idea to use icon color for status? Is dot color any better? :upside_down_face: Or we still use icon color for some other meaning?

Reply:
Makes sense, statusDot is fine. I'll change it. 

Applying it to the icon doesn't really work visually, I tried it. A good portion of the icon gets covered by the dot making it less recognisable and this way it is easier to see the dot.

The icon coloring still applies. It is used for the workflow status.

Reply:
![image](https://github.com/superdesk/superdesk-ui-framework/assets/2076953/bbe01829-fb90-4b46-aae6-00b555c50552)

![image](https://github.com/superdesk/superdesk-ui-framework/assets/2076953/b548a8c5-f941-4b68-90d0-782f1ce053db)


I'd do it something like that. Anyway, I'm leaving up to you to do the design it, just suggesting an idea.

Reply:
It's fine, I'm generally not against it. Semanticaly this does make more sense. But we are potentially getting into issues like this:
![Superdesk UI Framework 2024-07-05 at 1 52 43 PM](https://github.com/superdesk/superdesk-ui-framework/assets/1861461/5f91f283-0c99-4801-be57-b9f870938ba6)
I believe such combinations may not be possible, but I am not sure. If this assumption is correct, we could consider the second option with a slightly smaller dot. However, if the combinations with a colored icon and a dot do exist, I will relocate it to the top right corner.

Reply:
Yeah this is not ideal, but not that bad either. Since white outline exists it looks pretty well. And if we have a tooltip   describing the meaning, we're good I'd say.